python/looker_sdk/rtl/transport.py

```diff
@@ -77,7 +86,27 @@ class Response:
     """
 
     ok: bool
-    value: TResponseValue
+    value: bytes
+    response_mode: ResponseMode
+    encoding: str = "utf-8"
```

Comment:
This is the change to the interface - the `Response` object now has `value: bytes` and the additional `response_mode` and `encoding` properties

Reply:
Thanks for calling this out.